password-strength-checker/enabled.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {
+  name: 'the password strength checker is enabled',
+  hypothesis: 'the password strength checker will prevent users from signing up with insecure passwords.',
+  startDate: '2015-01-01',
+  subjectAttributes: ['uniqueUserId', 'isMetricsEnabled', 'forcePasswordStrengthCheck'],
```

Comment:
watch out for: https://github.com/mozilla/fxa-content-experiments/blob/dev/mailcheck/enabled.js

"isMetricsEnabledValue" vs "isMetricsEnabled"